content/docs/SUSHI/configuration/exhaustive-config.yaml

```diff
@@ -147,6 +163,10 @@ pages:
 # * useContext
 # * copyright
 # * packageId
+# The R5 ImplementationGuide resource also defines properties that
+# can be used as is not represented above:
```

Comment:
It was reading wrong for me (`as is not represented above`?)
```suggestion
# can be used as-is that are not represented above:
```

Reply:
Ah yeah that is confusing. I reworded it a little (basically just used the one above). Let me know what you think.

Reply:
I like yours even better!